Lib/warnings.py

```diff
@@ -483,6 +483,27 @@ def __exit__(self, *exc_info):
         self._module._showwarnmsg_impl = self._showwarnmsg_impl
 
 
+_DEPRECATED_MSG = "{name!r} is deprecated and slated for removal in Python {remove}"
```

Comment:
```suggestion
_DEPRECATED_MSG = "{name!r} is deprecated and slated for removal in Python {remove}."
```

Sorry, I could not resist. :)

Reply:
I could find anything in PEP 8 about ending exception/warning messages with a period and the code base is inconsistent.

Reply:
Never mind. Guido pointed out that most exception messages don't have a full stop, too. Let's stick to your approach.